src/helper/kzillaXYZ.ts

```diff
@@ -48,7 +48,7 @@ export async function handleShrinkURLMessage(
         createBasicEmbed(ERRORS.URL_MISSING, "ERROR")
       );
     }
-  } catch (err) {
+  } catch (err: any) {
```

Comment:
Why any?

Reply:
Oh no, changing it to `: Error`

Reply:
I'm pretty sure most of the places you had error as `any` datatype, I must've gone with that at the moment, anyway, changing that ASAP